solidity/coverage.sh

```diff
@@ -1,40 +1,40 @@
 set -e # exit on error
 
 # generates lcov.info
-forge coverage --report lcov
+forge coverage --report lcov && \
```

Comment:
I don't really understand why this is necessary and iirc it suppresses logs or something annoying
the `set -e` should exit if any command fails